test/Frontend.hs

```diff
@@ -0,0 +1,164 @@
+module Frontend where
```

Comment:
Eventually we should turn these into Tasty-style tests and add them to `test/Main.hs` so `stack test` works.

Reply:
I agree

https://github.com/typedefs/typedefs/blob/master/src/Typedefs/Test/ParseTests.idr#L25

Seems fairly good form wise. I'll see what i can do over time.

Need to derive equality for the syntax